invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ExportDropdown.js

```diff
@@ -26,6 +32,7 @@ export class ExportDropdown extends Component {
       <Grid>
         <Grid.Column width={11}>
           <Dropdown
+            aria-label="Export selection"
```

Comment:
We also probably want internationalization on the core module if we don't use it at NU:

```suggestion
            aria-label={i18next.t("Export selection")}
```